src/admin/components/UserResumeTable/index.tsx

```diff
@@ -79,11 +99,26 @@ const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
     {
       title: 'Planned Graduation',
       dataIndex: 'user.graduationYear',
+
+      filterMultiple: true,
+      filters: yearFilters, // [{ text: "2016", value: "2016" }],
+
+      onFilterDropdownVisibleChange: (visible) => {
+        if (visible) {
+          setTimeout(() => updateYearFilters());
+        }
+      },
+
+      // Value we are filtering by, record = content of current row
+      onFilter: (value: string, record: UserResume) => `${record.user.graduationYear}` === value,
     },
     {
       title: 'Date Uploaded',
       dataIndex: 'lastUpdated',
       render: (date: string) => <span>{reformatDate(date)}</span>,
+
+      onFilter: (value, record) => record.name.indexOf(value) === 0,
```

Comment:
What is this line of code used for? I don't think we have any filter applying to the "Date Uploaded" column.